src/api/fetchFolders.js

```diff
@@ -0,0 +1,18 @@
+export function fetchFolders(folderId) {
+  const url =
+    folderId === 'all'
+      ? 'https://bootcamp-api.codeit.kr/api/users/1/links'
+      : `https://bootcamp-api.codeit.kr/api/users/1/links?folderId=${folderId}`;
```

Comment:
```suggestion
  const query = folderId === 'all' ? '' : `?folderId=${folderId}`;
  const url = `https://bootcamp-api.codeit.kr/api/users/1/links${query}`;
```

이런식으로 query를 따로 관리하면 조금 더 깔끔하게 볼 수 있을 것 같아요.